cmd/generate-bindings/bindings/sourcecre.go.tpl

```diff
@@ -101,12 +101,21 @@ type {{$call.Normalized.Name}}Output struct {
 {{end}}
 
 // Events
-{{range $event := $contract.Events}}type {{.Normalized.Name}} struct {
+{{range $event := $contract.Events}}
+
+type {{.Normalized.Name}} struct {
 	{{- range .Normalized.Inputs}}
 	{{capitalise .Name}} {{if .Indexed}}{{bindtopictype .Type $.Structs}}{{else}}{{bindtype .Type $.Structs}}{{end}}
 	{{- end}}
 }
 
+// Decoded Events (indexed dynamic fields -> common.Hash (as you cannot decode from their hashes))
+type {{.Normalized.Name}}Decoded struct {
```

Comment:
I still don't get why we aren't replacing the other one, when would they be able to use it?